lambda/business-wallet/scripts/registerAndVote.js

```diff
@@ -0,0 +1,96 @@
+import env from "dotenv";
+import { invoke, invokeInBatch } from "../../lambdaUtils.mjs";
+import { solidityKeccak256 } from "ethers/lib/utils.js";
+import getDeveloperWallet, {
+  waitUntilTransactionMined,
+  getUserWallet,
+  sleep,
+  checkAPIKey,
+} from "../../../helpers/utils.mjs";
+
+async function main() {
+  env.config();
+
+  // Checks if the API_KEY is set in the .env file.
+  checkAPIKey();
+
+  const developerAddress = await getDeveloperWallet();
+
+  /* *************************************************************** Register Proposal *************************************************************** */
+  console.log(
+    "***************************************************** Register Proposal *****************************************************\n"
+  );
+  // Invokes the lambda function to register the user as Proposal.
+  console.log("Invoking lambda function to register proposal...\n");
+  const resultJson = await invokeInBatch(
+    [
+      {
+        call: {
+          function: {
+            name: "addProposal",
+            args: ["Proposal Name", "Proposal Description"],
+          },
+          reason: "Registering the proposal",
+          lambda: process.env.LAMBDA_ADDRESS,
+        },
+      },
+    ],
+    "Register Proposal"
+  );
+  console.log("Lambda invocation for registering proposal initiated: \n");
+
+  // Waits for the transaction to be mined.
+  await waitUntilTransactionMined(resultJson);
+  console.log(
+    "Lambda invocation for registering user completed: " +
+      resultJson.transactionHash +
+      "\n"
+  );
+
+  // Waits for 5 seconds.
+  sleep(5000);
+
+  /* *************************************************************** Stake And Vote *************************************************************** */
+  console.log(
+    "***************************************************** Stake And Vote *****************************************************\n"
+  );
+
+  // Invokes the lambda function to vote and stake.
+  console.log("Invoking lambda to state and vote for proposal...\n");
+  const resultJson2 = await invokeInBatch(
```

Comment:
why not merge both invocations to create a 3 size batch?

Reply:
just to make it easy to understand in step by step manner.

Reply:
You can add comments to make it clear. This will demonstrate the power of batch.

Reply:
done